components/fxa-client/src/internal/commands/mod.rs

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::convert::TryFrom;
+
+pub mod send_tab;
+pub use send_tab::SendTabPayload;
+
+use super::device::Device;
+use super::error::*;
+
+#[derive(Clone, Debug)]
+pub enum IncomingDeviceCommand {
```

Comment:
hrm - so after writing that comment about visibility, I'm a little confused by seeing `pub` here? Maybe `pub(crate)` better reflects what's going on?

Reply:
This is one of the things that's currently used by some example crates.